lib/unsplash/photo.rb

```diff
@@ -54,9 +53,8 @@ def find(id, width: nil, height: nil, crop_rect: nil)
       # @param orientation [String] Filter by orientation of the photo. Valid values are landscape, portrait, and squarish.
       # @return [Unsplash::Photo] An Unsplash Photo if count parameter is omitted
       # @return [Array] An array of Unsplash Photos if the count parameter is specified. An array is returned even if count is 1
-      def random(count: nil, categories: nil, collections: nil, featured: nil, user: nil, query: nil, width: nil, height: nil, orientation: nil)
+      def random(count: nil, collections: nil, featured: nil, user: nil, query: nil, width: nil, height: nil, orientation: nil)
```

Comment:
Would be good to remove `width` `height`. Also for the find photo and user endpoints too